# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: fbe7310ee5f3
# Full Hash: fbe7310ee5f35c9b523ce7c4f6a6887ece5a68b4
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1708768
# File Overlap Count: 1
# Description:
#   Bug 1708768 - Create a pref to defer gfxPlatformFontList initialization until first use, and enable this on macOS. r=lsalzman
#   
#   This gives the RegisterFonts thread more time to complete its work, so that
#   the main thread doesn't have to wait for it in InitFontList.
#   
# ==============================================================================

diff -r a01eaeed432a -r fbe7310ee5f3 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue May 18 11:10:42 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue May 18 11:10:43 2021 +0000
@@ -4617,6 +4617,15 @@
   value: true
   mirror: once
 
+- name: gfx.font-list.lazy-init.enabled
+  type: bool
+#if defined(XP_MACOSX)
+  value: true
+#else
+  value: false
+#endif
+  mirror: once
+
 # Whether to load fonts (e.g. Twemoji Mozilla) bundled with the application:
 #  -1 - Auto behavior based on OS version (currently, disables loading on
 #       "low-memory" Android devices)
